seed/api/base/urls.py

```diff
@@ -9,10 +9,12 @@
 from seed.api.v1.urls import urlpatterns as apiv1
 from seed.api.v2.urls import urlpatterns as apiv2
 from seed.api.v2_1.urls import urlpatterns as apiv2_1
+from seed.api.v3.urls import urlpatterns as apiv3
 
 urlpatterns = [
     # add flat urls namespace for non-conforming endpoints, ugh
     url(r'^v1/', include((apiv1, 'seed'), namespace='v1')),
     url(r'^v2/', include((apiv2, 'seed'), namespace='v2')),
     url(r'^v2.1/', include((apiv2_1, 'seed'), namespace='v2.1')),
+    url(r'^v3/', include((apiv3, 'seed'), namespace='v3')),
```

Comment:
yep, v3!

Reply:
Ah, I kept meaning to ask you - Did you want to call this v2.2 or v3? I think you were originally thinking of calling it v2.2 and I might've accidentally called it v3 in one of our meetings and just kept calling it that. 😅

Reply:
I know we have been saying v2.2. I'm fine with v3. It seems like this is going to be a big update so maybe a big version bump is worth it.